### ISSUE TITLE
Getting wrong predicted answer!

### ISSUE THREAD

**Issue by SandhyaSuryanarayana** [cdqa-suite/cdQA#319]
I'm using a customized CSV file (created using pdf_converter) and fined-tuned the model using a SQUAD like dataset(created using annotator) to build a QA  model.

I'm getting wrong answers for most of the questions. I did read another similar issue that was raised here and I did try varying the retriever_score_weight of .predict() method. It does not seem to work. But the correct answer is in the top 5 predictions when I set n_predictions to 5.

- I want to know if there is any way I can improve the accuracy of my predictions? Because I'm getting wrong answers only when I'm using the custom data. The accuracy is much better when I ask questions on bnp dataset. 

- Also, even when I ask the same question that I have used while fine tuning(Json file) the model I still get wrong answers. Is there anything I can do for that? 

This is the code that I'm executing.

![image](https://user-images.githubusercontent.com/21112063/70175788-5e8dfd00-16a5-11ea-92f3-67962af4d034.png)

![image](https://user-images.githubusercontent.com/21112063/70175573-fa6b3900-16a4-11ea-85e0-d67343bd7783.png)

![image](https://user-images.githubusercontent.com/21112063/70175676-2b4b6e00-16a5-11ea-8bde-32cac9c033e9.png)
 
Any help is really appreciated.

Thank & regards,
Sandhya

**Comment by nehabharambe** [comment]
I am also facing a similar issue, when I am using custom domain data to fine tune the model, the accuracy decreasing. For fine tuning, I am using json file generated from the cdqa-annotator. Is annotated dataset messing with the model?

**Comment by andrelmfarias** [comment]
Hi,
Could you please share the size of your datasets (i.e. number of question-answer pairs)
Did you try to fine-tune the model trained on SQuAD 1.1 or did you use the pre-trained BERT model (with no fine-tune on SQuAD) ?

**Comment by nehabharambe** [comment]
I used the model trained on SQuAD 1.1 and for one paragraph, there were around 3-4 question answer pair. I have attached my json file
[sample_cdqa-v1.1-2.zip](https://github.com/cdqa-suite/cdQA/files/3927859/sample_cdqa-v1.1-2.zip)

**Comment by nehabharambe** [comment]
Awating your response. I would appreciate if you can help me with this issue.

**Comment by andrelmfarias** [comment]
I think your training / fine-tunning data is too small. There are only 10 paragraphs and 38 QA pairs. If you compare with SQuAD, there are about 80k QA-pairs in the training set. The model is clearly over-fitting.
